src/frontend/App.tsx

```diff
@@ -11,12 +12,14 @@ import AirtableFieldOptions from './FieldOptions';
 import Separator from './Separator';
 
 const BaseStructure = () => {
-  // useBase will re-render the block whenever the base's configuration changes: this includes
-  // updates to names, descriptions and field options, as well as tables/fields being added or
-  // removed. This means the block will always show the latest structure.
-  const base = useBase();
+  console.log('base', base);
   const tables = base.tables;
 
+  useWatchable(base, ['schema'], () => {
+    console.log('base', base); // TODO Doesn't log anything when base is changed
```

Comment:
This is the real issue, nothing happens when the base changes. Meaning, the watcher doesn't work as it should, despite the code being identical to https://github.com/Airtable/apps-base-schema/blob/master/frontend/settings.js#L101